bokeh/models/layouts.py

```diff
@@ -549,6 +553,20 @@ def __init__(self, *args, **kwargs) -> None:
     The index of the active tab.
     """)
 
+class CanvasBox(LayoutDOM):
```

Comment:
Suggest it might better if this was just named `Canvas` and the fiddly name went to the lower level, e.g. `CanvasElement` or `CanvasDOM`, etc.

Reply:
This is a very early work in progress, so any naming convention changes I will leave for a later time, when the overall design will be more fleshed out.